examples/platforms/nrf52840/README.md

```diff
@@ -110,9 +122,7 @@ $ make -f examples/Makefile-nrf52840 DISABLE_CC310=1
 ## Flashing the binaries
 
 Flash the compiled binaries onto nRF52840 using `nrfjprog` which is
-part of the [nRF5x Command Line Tools][nRF5x-Command-Line-Tools].
-
-[nRF5x-Command-Line-Tools]: https://www.nordicsemi.com/eng/Products/nRF52840#Downloads
+part of the [nRF5 Command Line Tools][nRF5-Command-Line-Tools].
 
 ```bash
 $ nrfjprog -f nrf52 --chiperase --program output/nrf52840/bin/ot-cli-ftd.hex --reset
```

Comment:
Please apply changes from the 811 file to the Running the example section, including but not limited to:

To test the example:
1. Prepare two boards with the flashed `CLI Example` (as shown above).
2. As the CLI example uses UART connection, start a terminal
   emulator like PuTTY and connect to the used COM port with the following UART settings:
    - Baud rate: 115200
    - 8 data bits
    - 1 stop bit
    - No parity
    - HW flow control: RTS/CTS
   On Linux system a port name should be called e.g. `/dev/ttyACM0` or `/dev/ttyACM1`.
This allows you to view the raw UART output.
3. Open a terminal connection on the first board and start a new Thread network.
 ```bash
 > panid 0xabcd
 Done
 > ifconfig up
 Done
 > thread start
 Done
 ```
After a couple of seconds the node becomes a Leader of the network.
 ```bash
 > state
 Leader
 ```
4. Open a terminal connection on the second board and attach a node to the network.
 ```bash
 > panid 0xabcd
 Done
 > ifconfig up
 Done
 > thread start
 Done
 ```
After a couple of seconds the second node attaches and becomes a Child.
 ```bash
 > state
 Child
 ```
5. List all IPv6 addresses of the first board.
 ```bash
 > ipaddr
 fdde:ad00:beef:0:0:ff:fe00:fc00
 fdde:ad00:beef:0:0:ff:fe00:9c00
 fdde:ad00:beef:0:4bcb:73a5:7c28:318e
 fe80:0:0:0:5c91:c61:b67c:271c
 ```
8. Choose one of them and send an ICMPv6 ping from the second board.
 ```bash
 > ping fdde:ad00:beef:0:0:ff:fe00:fc00
 16 bytes from fdde:ad00:beef:0:0:ff:fe00:fc00: icmp_seq=1 hlim=64 time=8ms
 ```
For a list of all available commands, visit [OpenThread CLI Reference README.md][CLI].

Reply:
I don't fully understand this comment. I did some changes to align both instructions, however they are fundamentally different so i can't just copy/paste.

Reply:
I understand. My edit is more about how the steps are broken, not the contents of the procedure.